scylla/src/transport/session.rs

```diff
@@ -2176,18 +2173,6 @@ impl RequestSpan {
         }
     }
 
-    pub(crate) fn record_result_fields(&self, result: &LegacyQueryResult) {
-        self.span.record("result_size", result.serialized_size);
-        if let Some(rows) = result.rows.as_ref() {
-            self.span.record("result_rows", rows.len());
-        }
-    }
-
-    pub(crate) fn record_rows_fields(&self, rows: &Rows) {
-        self.span.record("result_size", rows.serialized_size);
-        self.span.record("result_rows", rows.rows.len());
-    }
-
     pub(crate) fn record_replicas<'a>(&'a self, replicas: &'a [(impl Borrow<Arc<Node>>, Shard)]) {
         struct ReplicaIps<'a, N>(&'a [(N, Shard)]);
```

Comment:
I only have serialized, raw frame contents:
```
------------------------------------
| metadata | raw rows  |
------------------------------------
at this point size of metadata and raw rows is unknown; we only know the size of them combined. Similarly, because we don't know where metadata ends and raw rows commence, we cannot deserialize row count.

Reply:
Instead of removing tracing of this totally can you trace the size of metadata + rows instead?

Reply:
Yes, I can trace the size of them combined - will change. I still have to have row count removed.